src/app/(app)/attendance/components/CurrentAttendance.tsx

```diff
@@ -0,0 +1,193 @@
+"use client";
+import { Button } from "@/components/ui/button";
+import { Card, CardContent } from "@/components/ui/card";
+import { Switch } from "@/components/ui/switch";
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from "@/components/ui/table";
+import { AttendanceStatus, MemberRole, Shift } from "@/constant/constant";
+import { User } from "@/types";
+import {
+  ColumnDef,
+  flexRender,
+  getCoreRowModel,
+  useReactTable,
+} from "@tanstack/react-table";
+import React, { useMemo, useState } from "react";
+
+export interface Attendance {
```

Comment:
We already have an interface defined as `Attendance` in `AttendanceList`. @ghulamhafeez @Mubashar1009 can help better.

Reply:
I have fixed this one by using common  `Attendance` interface.